Makefile

```diff
@@ -26,7 +26,10 @@ install:
 
 pep8:
 	@python -m pycodestyle lib/serviceAccessConfig/*.py
-	@python -m pycodestyle --ignore=E402 tests/unit/*.py 
+	@python -m pycodestyle --ignore=E402 tests/unit/*.py
```

Comment:
flake8 runs both pycodestyle and pyflakes. So calls direct to pycodestyle can be dropped as they are duplicative.